tsl/src/continuous_aggs/refresh.c

```diff
@@ -330,20 +330,10 @@ continuous_agg_refresh_with_window(const ContinuousAgg *cagg,
 	ExecDropSingleTupleTableSlot(slot);
 }
 
-#define REFRESH_FUNCTION_NAME "refresh_continuous_aggregate()"
-/*
- * Refresh a continuous aggregate across the given window.
- */
-Datum
-continuous_agg_refresh(PG_FUNCTION_ARGS)
+static ContinuousAgg *
+get_cagg_by_relid(Oid cagg_relid)
```

Comment:
Maybe `cagg_get_by_relid` ? to be in sync with function name, as `cagg_get_partition_type`, etc